### PR TITLE
[DJFS-40] Querying the transportation Database

### PR DESCRIPTION
**Pull request by ChabuChabu**
## Task: `Queried the database
- `Answered questions 1 to 5

### CODE REVIEW

File: submissions/chabuChabu/database/querydatabase.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*QUESTION  1*/
+use trips;
+select count(*) as survivors
+from survival
```
Comment (adaugochi) [inline submissions/chabuChabu/database/querydatabase.sql:4]: There is no table called `survival` from your previous tas

File: submissions/chabuChabu/database/querydatabase.sql [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+/*QUESTION 2 */
+select count(*) as died
+from survival
+where survived = 0;
+
+/*Answer: 549 passengers died*/
+
+/* QUESTION 3*/
+select passengers.Name,passengers.sex
+from passengers ,traveldetail
```
Comment (adaugochi) [inline submissions/chabuChabu/database/querydatabase.sql:18]: There is no table called `traveldetail` in your previous task

**Review by adaugochi — CHANGES_REQUESTED** [review]
The column names and table names didn't match what you did previously. It supposed to be a continuation of that

File: submissions/chabuChabu/database/trips.sql [hunk 3]
```diff
@@ -1,36 +1,56 @@
-/*The following query is used to create a table for storing passanger details*/
+/*QUESTION  1*/
 use trips;
```
Comment (adaugochi) [inline submissions/chabuChabu/database/trips.sql:2]: Why are you deleting all the things you did on the previous task

**Review by adaugochi — CHANGES_REQUESTED** [review]
Kindly check what you created as PR

File: submissions/chabuChabu/database/trips.sql (hunk 3 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/database/trips.sql:2]: Oh my word, thanks  for pointing that out  will re-submit

**Review by adaugochi — CHANGES_REQUESTED** [review]
You still haven't corrected yourself. Kindly click [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/pull/1160/files) to see the changes you are pushing as PR

**Comment by ChabuChabu** [comment]
Will push another PR

**Pull request closed without merging**